src/PhpHash.php

```diff
@@ -1,12 +1,14 @@
 <?php
 namespace Aws;
 
+use HashContext;
+
 /**
  * Incremental hashing using PHP's hash functions.
  */
 class PhpHash implements HashInterface
 {
-    /** @var resource */
+    /** @var HashContext */
```

Comment:
This should be `resource|HashContext` since it may also be a `resource` type for older versions of PHP. The same change should apply to the `@return` annotation on [Line 68](https://github.com/aws/aws-sdk-php/pull/1465/files#diff-9870abb8d621b0ea2887e4ba5cc617b7R68)